pkg/setting/setting.go

```diff
@@ -205,6 +205,9 @@ type Cfg struct {
 	RouterLogging    bool
 	Domain           string
 	CDNRootURL       *url.URL
+	ReadTimeout      time.Duration
+	EnableGzip       bool
+	EnforceDomain    bool
```

Comment:
Are these added by accident? I guess they have no effect, but might be better to remove them from the PR.

Reply:
ah yea!. They showed up as belong to the same commit after solving the merge conflict in #31575

Reply:
Fine to keep em? as the cherry-pick makes more sense that way.

Reply:
OK, I guess it's not important.